e2e/runner_test.go

```diff
@@ -184,3 +192,39 @@ func parseFeatureFlags(flags []string) ([]string, []*contextMetadata) {
 
 	return featurePaths, metadatas
 }
+
+// startRuntimeDependencies spins up the runtime dependencies for a stack, represented
+// by a docker-compose file, It will panic if they cannot be satisfied
+func startRuntimeDependencies(stackName string, env map[string]string, minutesToBeHealthy time.Duration) {
+	serviceManager := services.NewServiceManager()
+
+	err := serviceManager.RunCompose(true, []string{stackName}, env)
+	if err != nil {
+		log.WithFields(log.Fields{
+			"stack": stackName,
+		}).Error("Could not run the stack.")
+		panic("Could not run the stack.")
+	}
+
+	healthy, err := waitForElasticsearch((minutesToBeHealthy * time.Minute))
+	if !healthy {
+		log.WithFields(log.Fields{
+			"error":   err,
+			"minutes": minutesToBeHealthy,
+			"stack":   stackName,
+		}).Error("The Elasticsearch cluster could not get the healthy status")
+		panic("The Elasticsearch cluster could not get the healthy status")
+	}
+}
+
+// tearDownRuntimeDependencies destroys the runtime dependencies for a stack,
+// not failing the execution in the case it's not possible to destroy them
+func tearDownRuntimeDependencies(stackName string) {
```

Comment:
Should we keep `stackName` here? Looks like it has been removed from other similar functions?

Reply:
Yes, we should keep it to be able to execute docker-compose down to the proper compose file.

We could think about having some state to simply use the compose we initially ran, maybe in a follow-up?